examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1424 @@
+{
```

Comment:
<p>In fact we also support middle-out approach </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Yes, but in the current implementation it is not obvious. We don't provide separate functionality for this method either.</p>

Reply:
<p>Explain how to use is somewhere because somebody might think that it can't be done at all</p>